Makefile

```diff
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 
-BUNDLE_VERSION ?= 4.7.13
+BUNDLE_VERSION ?= 4.8.0-rc.0
```

Comment:
This version update is only to test if current master along with this PR is able to provide us a green CI or do we still need some more changes to incorporate 4.8 release with crc.